18F/identity-idp
# LG-6696: FSMv2: Localize API requests with current locale

**Why**: So that the API controller logic uses the locale corresponding to the user's current UI preference.

In particular, the `Api::Verify::PasswordResetController` controller is expected to [respond with a redirect URL](https://github.com/18F/identity-idp/blob/46182cf89bc8d0354ca6c6c662c04f2a6a26dea9/app/controllers/api/verify/password_reset_controller.rb#L12). Previously, this would always respond with the English URL, regardless of the user's preferred language.

**Testing Instructions:**

1. Go to http://localhost:3000
2. Sign in
3. Go to http://localhost:3000/verify
4. Complete proofing flow up to "Re-enter your password" step
5. Change page language to Spanish or French
6. Click the "Forgot your password?" link, as translated to the current language
7. Click the "Reset password" button, as translated to the current language (it is the secondary outline button)

_Before:_ Password reset confirmation page is shown in English.
_After:_ Password reset confirmation page is shown in the current language.

**Screenshot:**

![Screen Shot 2022-06-24 at 8 53 51 AM](https://user-images.githubusercontent.com/1779930/175542255-c96870fa-69db-4106-a48a-81eee0c3a567.png)
